src/reference/antora/modules/ROOT/pages/mail.adoc

```diff
@@ -181,32 +181,42 @@ The framework provides a `MailToStringTransformer` that you can use to convert t
 
 [tabs]
 ======
-Java DSL::
+Java::
 +
 [source, java, role="primary"]
+----
+@Bean
+@Transformer(inputChannel="...", outputChannel="...")
+public Transformer transformer() {
+    return new MailToStringTransformer();
+}
+----
+
+Java DSL::
++
+[source, java, role="secondary"]
 ----
    ...
    .transform(Mail.toStringTransformer())
    ...
 ----
 
-Java::
+Kotlin DSL::
 +
-[source, java, role="secondary"]
+[source, kotlin, role="secondary"]
 ----
-@Bean
-@Transformer(inputChannel="...", outputChannel="...")
-public Transformer transformer() {
-    return new MailToStringTransformer();
-}
+   ...
+   .transform(Mail.toStringTransformer())
```

Comment:
Just minor: there is no need in period for Kotlin/Groovy DSLs.
See examples in the `KotlinDslTests` and `GroovyDslTests`, respectively.
😄